jbpm-workitems/pom.xml

```diff
@@ -104,4 +109,14 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.testing.compile</groupId>
+        <artifactId>compile-testing</artifactId>
+        <version>0.11</version>
```

Comment:
@psiroky what do you think, is it ok to have it declared like this or do we need to add it to one of boms?

Reply:
Generally, this should go into `kie-parent`.

Reply:
@psiroky could you point me where exactly this goes for master and 7.3.x branches in kie-parent so i don't make some mistake ? Thanks

Reply:
Exactly here https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/9dd791726c9a29e1cbdc9846b936f6c89854d95a/pom.xml#L1807. Ideally with the version extracted into a property `version.com.google.testing.compile`. Same for 7.3.x